tabs/src/components/ui/prototypes/Chat.jsx

```diff
@@ -0,0 +1,135 @@
+import React from "react";
+import { 
+  AcceptIcon,
+  Avatar,
+  Box,
+  Button,
+  Chat as ChatTag,
+  ChatItemProps,
+  Header,
+  MentionIcon,
+  RedbangIcon,
+  Segment, 
+  ShorthandCollection } from '@fluentui/react-northstar'
+import ChatMessagesCode from '!!raw-loader!../../code/prototypes/chat/ChatMessagesCode.jsx'
```

Comment:
will this be able to use Asset module?
https://webpack.js.org/guides/asset-modules/
it seems !!raw-loader! will be DEPREACTED for v5

Reply:
Thanks for your advice. I will learn how to use Asset modules in my project.

Reply:
Since our scaffolding used webpack v4, I will keep using raw-loader.